recipes/parmed/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set name = "ParmEd" %}
+{% set version = "2.7.3" %}
+{% set md5 = "027cd3b738318c044164ff5ca06295e5" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: ParmEd-{{ version }}.tar.gz
+  url: https://github.com/{{ name }}/{{ name }}/archive/{{ version }}.tar.gz
+  md5: {{ md5 }}
+
+
+build:
+  number: 1
+  script: python setup.py install
+
+requirements:
+  build:
+    - python
+    - numpy
+    - setuptools
+
+  run:
+    - python
+    - numpy
+    - pandas
+
+test:
+  imports:
+    - parmed
```

Comment:
Maybe add the submodules as well:

https://github.com/ParmEd/ParmEd/blob/master/setup.py#L63-L68